components/properties/src/props.rs

```diff
@@ -705,6 +739,7 @@ impl BidiClass {
     /// (`PDI`) U+2069: terminates an isolate control
     pub const PopDirectionalIsolate: BidiClass = BidiClass(22);
 }
+}
```

Comment:
I dont if I am right here, but I think you should indent here. I guess this won't change from `cargo fmt`

Reply:
I think keep it as-is. Unfortunately `rustfmt` doesn't yet support formatting inside macros. https://github.com/rust-lang/rustfmt/issues/8

Reply:
Thanks for referencing the underlying issue